plugins/BEdita/API/src/Controller/JsonBaseController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2022 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+namespace BEdita\API\Controller;
+
+use BEdita\API\Controller\AppController;
+
+/**
+ * Base class for controllers handling pure `application/json` content-type, not using JSON API
+ *
+ */
+abstract class JsonBaseController extends AppController
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function initialize(): void
+    {
+        parent::initialize();
+        if ($this->components()->has('JsonApi')) {
+            $this->components()->unload('JsonApi');
+        }
+        $this->viewBuilder()->setClassName('Json');
+        $this->RequestHandler->setConfig('viewClassMap.json', 'Json');
+
+        if ($this->request->contentType() === 'application/json') {
+            $this->RequestHandler->setConfig('inputTypeMap.json', ['json_decode', true], false);
```

Comment:
Why don't  we always set this config?

Reply:
In `AppController` we have 

```php
        if ($this->request->is(['json', 'jsonapi'])) {
            ....
            $this->RequestHandler->setConfig('inputTypeMap.json', [[$this->JsonApi, 'parseInput']], false);
            ....
       }
```
But then we unload `JsonApi` component - I thought that a config update is safer